groundstation/backend_api/user.py

```diff
@@ -28,12 +28,21 @@ def get(self, user_id, local_args=None):
         :returns: response_object, status_code
         :rtype: tuple (dict, int)
         """
-        user = User.query.filter_by(id=user_id).first()
+        user = User.query.filter_by(id=User.decode_auth_token(user_id)).first()
+        #userID = User.decode_auth_token(local_args['auth_token'])
+
         response_object = {
             'status':None,
             'message':None,
             'data':None
         }
+
+        # if user_id != userID:
+        #     response_object['status'] = 'fail'
+        #     response_object['message'] = 'User ID does not match auth token.'
+        #     response_object['data'] = None
+        #     return response_object, 404
+
```

Comment:
```suggestion
        response_object = {
            'status':None,
            'message':None,
            'data':None
        }
```
Since we switched to passing tokens to get user data, I believe it is safe to delete these commented out code blocks